src/components/screens/SendScreen/screens/TransactionProcessingScreen.tsx

```diff
@@ -179,9 +189,17 @@ const TransactionProcessingScreen: React.FC<
 
             <View className="rounded-[16px] p-[24px] gap-[24px] bg-background-secondary w-full">
               <View className="flex-row items-center justify-center gap-[16px]">
-                {selectedBalance && (
+                {type === "token" && selectedBalance && (
                   <TokenIcon token={selectedBalance} size="lg" />
                 )}
+                {type === "collectible" && selectedCollectible && (
```

Comment:
could you please review all places that could make use of the new enum types this PR is adding? thanks!